src/app/(fumadocs)/mdx-components.tsx

```
@@ -233,6 +302,48 @@
+			const { id, ...otherProps } = combinedProps;
+			return <Pricing_3 id={id ?? undefined} {...otherProps} />;
+		},
+		// UI Components
+		Card: (props: React.ComponentProps<"div">) => <Card {...props} />,
+		CardHeader: (props: React.ComponentProps<"div">) => (
+			<CardHeader {...props} />
+		),
+		CardTitle: (props: React.ComponentProps<"div">) => <CardTitle {...props} />,
+		CardDescription: (props: React.ComponentProps<"div">) => (
+			<CardDescription {...props} />
+		),
```

ESLint

[error] Expected "CardDescription" to come before "CardTitle".

(perfectionist/sort-objects)